pkg/driver/driver_azure.go

```diff
@@ -167,7 +167,7 @@ func (d *AzureDriver) getVMParameters(vmName string, networkInterfaceReferenceID
 		VMParameters.Identity = &compute.VirtualMachineIdentity{
 			Type: compute.ResourceIdentityTypeUserAssigned,
 			UserAssignedIdentities: map[string]*compute.VirtualMachineIdentityUserAssignedIdentitiesValue{
-				*d.AzureMachineClass.Spec.Properties.IdentityID: &compute.VirtualMachineIdentityUserAssignedIdentitiesValue{},
+				*d.AzureMachineClass.Spec.Properties.IdentityID: {},
```

Comment:
Have you tested this change? Does it work? And why is it needed? I ask this for my understanding.

Reply:
The -s flag (simplified) of gofmt is preferred. Ref https://golang.org/cmd/gofmt/#hdr-The_simplify_command
```
gofmt -l -w -s pkg/driver/driver_azure.go
```